common/static/common/css/base.css

```diff
@@ -181,14 +174,16 @@ body {
     margin-top: 11px !important;
 }
 .navbar {
-    background: url(http://eyebrowse.csail.mit.edu/static/common/img/bg-nav.png) repeat top left;
+    background: #f5f5f5;
 }
 .icon {
     width: 16px;
     height: 16px;
 }
 .well {
-    background-color: #DDD !important;
+    border: none;
+    box-shadow: none;
+    -webkit-box-shadow: none;
```

Comment:
sweet i'm going to go through and clean all of this up. Any other CSS advice is much appreciated :D